Data-Science/MLflow/bike-sharing-mlflow.ipynb

```diff
@@ -46,7 +104,7 @@
    "source": [
     "# Set up an experiment with set_exp from ezmllib.mlflow\n",
     "experiment_name = 'bike-sharing-exp'\n",
-    "mlflow.set_experiment(experiment_name)"
```

Comment:
Here also we don't need to import mlfow function it should be available in all notebook images, use mlflow.set_experiment directly.